php-practice/section6.php

```diff
@@ -132,9 +143,23 @@ function getSuperpower($superheroName) {
  */
 
 function getSuperpowerAlternate($superheroName) {
-  // YOUR CODE HERE
+  switch ($superheroName) {
+    case 'Superman':
+      return 'Super Strength';
+      break;
+    case 'Green Lantern':
+      return 'Magical Ring';
```

Comment:
So one interesting thing to note is, you forgot the `break` statement for this `case`. But `return` makes it stop executing the function and just return a value. So the behavior is the same as if you did a `break`.